docs/data/toolpad/core/components/app-provider/AppProviderBasic.tsx

```diff
@@ -38,7 +39,11 @@ export default function AppProviderBasic() {
 
   return (
     // preview-start
-    <AppProvider navigation={NAVIGATION} router={router}>
+    <AppProvider
+      navigation={NAVIGATION}
```

Comment:
In the navigation I also see
```
// Add the following new item:
```
I presume we don't need that comment in the context of this demo?